languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -83988,6 +83988,29 @@ USA
             <example correction="must|have to">I <marker>must to</marker> go shopping.</example>
             <example>It is a must to be aware of that.</example>
         </rule>
+        <rule id="TO_AFTER_MODAL_VERBS" name="to after modal verbs" default="temp_off">
+         <antipattern>
+            <token regexp="yes">an?</token>
+            <token regexp="yes" min="0">absolute|total</token>
+            <token regexp="yes">can|could|may|might|should|shall</token>
+         </antipattern>
+         <antipattern>
+            <token regexp="yes">that|(what|which|who)(ever)?</token>
```

Comment:
You can extend this token to include `all|everything|anything`
This will remove a lot of FPs like _We will do everything we can to assist_